cognite/neat/rules/models/information/_converter.py

```diff
@@ -111,6 +99,26 @@ def as_dms_architect_rules(self, created: datetime | None = None, updated: datet
             reference=self.information.reference.as_dms_architect_rules() if self.information.reference else None,
         )
 
+    @classmethod
+    def _convert_metadata_to_dms(cls, metadata: InformationMetadata) -> "DMSMetadata":
+        from cognite.neat.rules.models.dms._rules import (
+            DMSMetadata,
+        )
+
+        space = cls._to_space(metadata.prefix)
+
+        return DMSMetadata(
+            schema_=metadata.schema_,
+            space=space,
+            data_model_type=metadata.data_model_type,
+            version=metadata.version,
+            external_id=metadata.name.replace(" ", "_").lower(),
```

Comment:
this works as long as someone does not use other characters in the name such as ":" , etc.

Maybe it is safe to have one more regex replacement for any non-complinat characters